### PR TITLE
Organizations tests

### PR DESCRIPTION
**Pull request by amyrebecca**
Here are a whole heap of unit tests I wrote today.

### CODE REVIEW

**Comment by amyrebecca** [comment]
depends on #21

**Comment by amyrebecca** [comment]
This is now ready to merge IMO and the sooner the better because of some of the reorganization it does.

File: test/reducers-test.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    expect(abState.A).to.be.true;
+    expect(abState.B).to.be.true;
+    expect(abState.C).to.be.undefined;
+    expect(abState.D).to.be.true;
+  });
+
+  it('should be upset if multiple reducers match the same action', () => {
+    expect(sampleReducers.bind(null, initialState, { type: 'C' })).to.throw('Multiple reducers matched the action');
+  });
+});
```
Comment (mcbouslog) [inline test/reducers-test.js]: very cool! helped me understand reducers more.

File: test/utils/dom.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+var doc = jsdom.jsdom('<!doctype html><html><body></body></html>');
+
+// get the window object out of the document
+var win = doc.defaultView;
+
+// set globals for mocha that make access to document and window feel
+// natural in the test environment
+global.document = doc;
+global.window = win;
+
```
Comment (amyrebecca) [inline test/utils/dom.js]: This is necessary to get enzyme's full DOM rendering working and you can find it in their docs. Keep in mind this only affects when we run the tests.

**Pull request merged**